tests/all/wasm_opt.rs

```diff
@@ -147,7 +147,7 @@ fn custom_args() {
         .arg("build")
         .assert()
         .stderr(predicates::str::contains("--not-accepted-argument"))
-        .failure();
```

Comment:
this pr changes the expected behavior so i changed this failure to a success. i've talked with @ashleygwilliams about this change and we think it makes sense to build and warn if `wasm-opt` fails since it's not a necessary build step. That being said, I could see how you might want to make it fail instead of warn if it was custom arguments that failed and not the default `wasm-opt` enablement that failed.

cc @alexcrichton - this is related to https://github.com/rustwasm/wasm-pack/pull/625

Reply:
looks good to me